src/main/java/com/nvidia/spark/rapids/jni/HyperLogLogPlusPlusHostUDF.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2024, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids.jni;
+
+import ai.rapids.cudf.ColumnVector;
+import ai.rapids.cudf.ColumnView;
+import ai.rapids.cudf.NativeDepsLoader;
+
+/**
+ * HyperLogLogPlusPlus(HLLPP) host UDF aggregation utils
+ */
+public class HyperLogLogPlusPlusHostUDF {
```

Comment:
Now the Java interface has changed. Please reimplement this similar to https://github.com/NVIDIA/spark-rapids-jni/pull/2631/files#diff-3bf8ba05afd52e4ef36fa2c0431304bbc88bc07cafd976665e17113464811392R24-R41.

Reply:
done